client/me/purchases/manage-purchase/index.jsx

```diff
@@ -180,8 +180,11 @@ class ManagePurchase extends Component {
 	}
 
 	handleRenew = () => {
-		const { purchase, siteSlug, redirectTo } = this.props;
-		const options = redirectTo ? { redirectTo } : undefined;
+		const { purchase, siteSlug, redirectTo, productsList } = this.props;
+
+		const options = { redirectTo };
```

Comment:
This seems to change the behavior of the code before, which checks for `redirectTo` before it gets added. I don't know if that would make any difference. But would that be a safer alternative?

```
const options = redirectTo ? { redirectTo } : {};
```

Reply:
For this case, I think it wouldn't be a problem since `options` has a default value of `{}` in `handleRenewNowClick`:

https://github.com/Automattic/wp-calypso/blob/78d74cca5c6f8ac75d667f435ebfbfb3c5281427/client/lib/purchases/index.js#L138

Also, there's a check for the `redirectTo` prop later on:

https://github.com/Automattic/wp-calypso/blob/78d74cca5c6f8ac75d667f435ebfbfb3c5281427/client/lib/purchases/index.js#L163

Reply:
Good point. I _think_ it's safe, but for other reasons I've reverted this change anyway,